staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

```diff
@@ -68,24 +68,24 @@ type JSONSchemaURL string
 // JSONSchemaPropsOrArray represents a value that can either be a JSONSchemaProps
 // or an array of JSONSchemaProps. Mainly here for serialization purposes.
 type JSONSchemaPropsOrArray struct {
-	Schema      *JSONSchemaProps  `protobuf:"bytes,1,opt,name=schema"`
-	JSONSchemas []JSONSchemaProps `protobuf:"bytes,2,rep,name=jSONSchemas"`
+	Schema      *JSONSchemaProps  `json:"Schema,omitempty" protobuf:"bytes,1,opt,name=schema"`
+	JSONSchemas []JSONSchemaProps `json:"JSONSchemas,omitempty" protobuf:"bytes,2,rep,name=JSONSchemas"`
```

Comment:
this will cause a breaking change for proto

Reply:
this will also make these fields optional (instead of required) as before.

Reply:
`jsonSchemas`